src/main/java/net/mcreator/ui/blockly/BlocklyPanel.java

```diff
@@ -100,7 +102,13 @@ public BlocklyPanel(MCreator mcreator) {
 						css += FileIO.readResourceToString("/blockly/css/mcreator_blockly_unixfix.css");
 					}
 
-					css += FileIO.readResourceToString("/blockly/css/" + UIManager.get("MCreatorLAF.BLOCKLY_CSS"));
+					// We use by default the dark color theme
+					String id = ThemeLoader.DARK_THEME.getID();

```

Comment:
I think all themes could use the same code here, no need to differentiate for dark theme. Also consider dropping MCreatorLAF.BLOCKLY_CSS as all themes can now have same file blockly_style.css, as they are differentiated by folder where this file is in. Same for codeeditor_dark.xml, can be codeditor_colors.xml for all themes.

Reply:
@Goldorion this is still unresolved. I mean that all themes would have file named the same, as folder separates them, so:

* mcreator-themes/theme1/codeditor_colors.xml
* mcreator-themes/theme2/codeditor_colors.xml

Similar for css files

Reply:
It should be ok now.